### PR TITLE
Add RedisCacheManager section to caching.md

### PR DESCRIPTION
**Pull request by alexxiyang**
Add RedisCacheManager section to caching.md

### CODE REVIEW

**Comment by bdemers** [comment]
Can you add this to `integration.md` instead please :)

**Comment by alexxiyang** [comment]
my project is already in ‘ https://shiro.apache.org/integration.html‘
:
Redis Cache Manager from @alexxiyang
A Redis Cache Manager implementation.

I just want to add a placeholder in caching.md so that there is an implement of redis. 

How about We put a placeholder *RedisCacheManager* and write down TBD like *EhCacheManager*. And append with the link of integration.md without mention about my project?

**Comment by alexxiyang** [comment]
I've remove shiro-redis from caching.md. Just leave the RedisCacheManager with integration.html link.

**Comment by alexxiyang** [comment]
Thank you for your suggestion @bdemers . I didn't fully understand "and add an anchor above the Redis and Memcached items" . I've changed the caching page. Could you help to review it again? 

Thanks

File: caching.md [hunk 1]
```diff
@@ -64,6 +64,11 @@ TBD
 
 TBD
 
+<a name="Caching-%7B%7BOtherCommunitySupportedCacheManagers%7D%7D"></a>
```
Comment (bdemers) [inline caching.md]: This anchor isn't needed, Most of the manually defined anchors are for backwards compatibility (the original site was generated/exported through confluence)

File: caching.md [hunk 2]
```diff
@@ -64,6 +64,11 @@ TBD
...
+###`Other Community Supported Cache Managers`
+
+[Integration](integration.html "Integration")
```
Comment (bdemers) [inline caching.md]: This works for me, but to fix the confusion about my comment about adding an anchor.

You could create a new anchor on the integration page between `Lift` and `Redis Cache Manager` called something like `cache-managers`, then change this link to:
```
[Integration](integration.html#cache-managers "Integration")
```

But, that isn't strictly needed :)

**Comment by alexxiyang** [comment]
@bdemers I've checked in some changes based on your comments

**Pull request closed without merging**